backend/de.metas.business/src/main/java/de/metas/shippingnotification/ShippingNotificationLoaderAndSaver.java

```diff
@@ -158,7 +170,7 @@ private ArrayList<I_M_Shipping_NotificationLine> retrieveLineRecords(final @NonN
 				.collect(Collectors.toCollection(ArrayList::new));
 	}
 
-	public Map<ShippingNotificationId, ArrayList<I_M_Shipping_NotificationLine>> getLineRecords(@NonNull final Set<ShippingNotificationId> ids)
+	public Map<ShippingNotificationId, ArrayList<I_M_Shipping_NotificationLine>> getLineRecords(@NonNull final Collection<ShippingNotificationId> ids)
```

Comment:
@cp-ps  for future,
Map<ShippingNotificationId, ArrayList<I_M_Shipping_NotificationLine>>
could be replaced by ArrayListMultimap